README_zh_CN.md

```diff
@@ -74,33 +77,88 @@ service minio start
 ```
 
 ## 使用源码安装
+源码安装仅供开发人员和高级用户使用,如果你还没有可以工作的Golang环境， 请参考 [How to install Golang](https://docs.min.io/cn/how-to-install-golang).
+
+```sh
+GO111MODULE=on go get github.com/minio/minio
+```
 
-采用源码安装仅供开发人员和高级用户使用,如果你还没有Golang环境， 请参考 [How to install Golang](https://docs.min.io/docs/how-to-install-golang).
+## 允许防火墙端口访问  
+在默认情况下，MinIO使用9000端口监听进来的连接，如果你的平台默认阻断端口，你可能需要通过设置使其能访问端口。
+
+### iptables
+对于拥有激活iptable（像RHEL,CentOs等等）的主机而言，你能够通过`iptables`命令使全部的通讯都连接到特定的端口。使用下面的命令来准许访问9000端口
 
 ```sh
-go get -u github.com/minio/minio
+iptables -A INPUT -p top -dport 9000 -j ACCEPT
+service iptalbes restart
+```
+
+下面的命令可以使全部接入的通讯连接到从9000到9010的端口。
+
+```sh
+iptables -A INPUT -p tcp -dport 9000:9010 -j ACCEPT
+service iptables restart
+```
+
+### ufw
+
+对于拥有激活ufw（基于distros的Debian）的主机而言，你可以使用`ufw`命令来准许通讯访问特定的端口，使用下面的命令来准许访问9000端口
+
+```sh
+ufw allow 9000
+```
+
+下面的命令可以使全部接入的通讯连接到从9000到9010的端口。
+
+```sh 
+ufw allow 9000:9010/tcp
+```
+
+### 防火墙命令(firewall-cmd)  
+对于拥有激活firewall-cmd（CentOS）的主机而言，你能够使用`firewall-cmd`命令来准许通讯连接到特定的端口。使用下面的命令来准许访问9000端口
+
+```sh
+firewall-cmd -get-active-zones
+```
+
+这个命令可以激活zone（s）。现在，应用端口规则到返回以上的相关zones中。例如，如果zone是`public`,使用
+
+```sh
+firewall-cmd -zone=public -add-port=9000/tcp -permanent
+```
+
+注意参数`permanent`可以确保规则持续存在于防火墙的启动、重启或重载之中。最后重载防火墙来使更改生效。
+
+```sh
+firewall-cmd -reload
 ```
 
 ## 使用MinIO浏览器进行验证
-安装后使用浏览器访问[http://127.0.0.1:9000](http://127.0.0.1:9000)，如果可以访问，则表示minio已经安装成功。
+
+MinIO服务具有一个嵌入的基于对象浏览器的网站。点击你的浏览器访问http://127.0.0.1:9000，确保你的服务器已经成功启动。
 
 ![Screenshot](https://github.com/minio/minio/blob/master/docs/screenshots/minio-browser.png?raw=true)
 
 ## 使用MinIO客户端 `mc`进行验证
-`mc` 提供了一些UNIX常用命令的替代品，像ls, cat, cp, mirror, diff这些。 它支持文件系统和亚马逊S3云存储服务。 更多信息请参考 [mc快速入门](https://docs.min.io/docs/minio-client-quickstart-guide) 。
+`mc` 提供了UNIX常用命令的替代品，像ls, cat, cp, mirror, diff等。 它支持文件系统和亚马逊S3云存储服务。 更多信息请参考 [mc快速入门](https://docs.min.io/docs/minio-client-quickstart-guide) 。
 
 ## 已经存在的数据
-当在单块磁盘上部署MinIO server,MinIO server允许客户端访问数据目录下已经存在的数据。比如，如果MinIO使用`minio server /mnt/data`启动，那么所有已经在`/mnt/data`目录下的数据都可以被客户端访问到。
+当在单块磁盘上部署MinIO server,MinIO server允许客户端访问数据目录下已经存在的数据。比如，如果使用`minio server /mnt/data`启动MinIO，那么所有已经在`/mnt/data`目录下存在的数据都可以被客户端访问到。
 
 上述描述对所有网关后端同样有效。
 
 ## 了解更多
-- [MinIO纠删码入门](https://docs.min.io/docs/minio-erasure-code-quickstart-guide)
-- [`mc`快速入门](https://docs.min.io/docs/minio-client-quickstart-guide)
-- [使用 `aws-cli`](https://docs.min.io/docs/aws-cli-with-minio)
+- [MinIO纠删码入门](https://docs.min.io/cn/minio-erasure-code-quickstart-guide)
+- [`mc`快速入门](https://docs.min.io/cn/minio-client-quickstart-guide)
+- [使用 `aws-cli`](https://docs.min.io/cn/aws-cli-with-minio)
 - [使用 `s3cmd`](https://docs.min.io/docs/s3cmd-with-minio)
```

Comment:
This link also needs to be replaced with its Chinese version, right?
`https://docs.min.io/docs/s3cmd-with-minio` => `https://docs.min.io/cn/s3cmd-with-minio`